shell/platform/darwin/common/framework/Source/FlutterBinaryMessengerRelayTest.mm

```diff
@@ -2,49 +2,48 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterBinaryMessengerRelay.h"
+#import "flutter/shell/platform/darwin/common/framework/Source/FlutterBinaryMessengerRelay.h"
 
 #import <OCMock/OCMock.h>
-#import <XCTest/XCTest.h>
 
 #import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+#import "flutter/shell/platform/darwin/common/framework/Source/FlutterTestUtils.h"
+#import "flutter/testing/testing.h"
+#include "gtest/gtest.h"
 
 FLUTTER_ASSERT_ARC
 
 @protocol FlutterTaskQueue <NSObject>
 @end
 
-@interface FlutterBinaryMessengerRelayTest : XCTestCase
+@interface FlutterBinaryMessengerRelayTest : NSObject
 @end
 
 @implementation FlutterBinaryMessengerRelayTest
 
-- (void)setUp {
-}
-
-- (void)tearDown {
-}
-
-- (void)testCreate {
+- (BOOL)testCreate {
   id messenger = OCMProtocolMock(@protocol(FlutterBinaryMessenger));
   FlutterBinaryMessengerRelay* relay =
       [[FlutterBinaryMessengerRelay alloc] initWithParent:messenger];
-  XCTAssertNotNil(relay);
-  XCTAssertEqual(messenger, relay.parent);
+  EXPECT_NE(relay, nil);
+  EXPECT_EQ(messenger, relay.parent);
+  return YES;
 }
 
-- (void)testPassesCallOn {
+- (BOOL)testPassesCallOn {
   id messenger = OCMProtocolMock(@protocol(FlutterBinaryMessenger));
   FlutterBinaryMessengerRelay* relay =
       [[FlutterBinaryMessengerRelay alloc] initWithParent:messenger];
   char messageData[] = {'a', 'a', 'r', 'o', 'n'};
   NSData* message = [NSData dataWithBytes:messageData length:sizeof(messageData)];
   NSString* channel = @"foobar";
   [relay sendOnChannel:channel message:message binaryReply:nil];
-  OCMVerify([messenger sendOnChannel:channel message:message binaryReply:nil]);
+  return !FLTThrowsObjcException(^{
+    OCMVerify([messenger sendOnChannel:channel message:message binaryReply:nil]);
+  });
```

Comment:
I was thinking moving `FLTThrowsObjcException` up one level higher, then all of these tests can stay unchanged.

Reply:
Ah yeah - much better!